Makefile

```diff
@@ -2,13 +2,33 @@
 
 REPO   := docker.stackable.tech/stackable
 TAG    := $(shell git rev-parse --short HEAD)
+ARCH   := $(shell arch)
 
 define push
 	docker push --all-tags ${REPO}/$(1)
 endef
 
 define build
-	@docker build --force-rm -t "${REPO}/${1}:${TAG}" -t "${REPO}/${1}:latest" -f $(1)/Dockerfile .
+	@docker build --force-rm -t "${REPO}/${1}:${TAG}-${ARCH}" -t "${REPO}/${1}:latest-${ARCH}" -f $(1)/Dockerfile .
+endef
+
+# Tag for arm64 is fixed since this will run on gh ubuntu-latest
+define manifest
+	@docker manifest create "${REPO}/${1}:latest" ${REPO}/${1}:latest-${ARCH} ${REPO}/${1}:latest-aarch64
```

Comment:
Don't love the racy way that this works... Isn't there a way for us to save the build hashes between the steps?

Reply:
Why a racy way?

Reply:
It assumes that nothing else writes to `latest-$arch` in between the steps.

Reply:
Ah yeah. The way the pipeline is designed now (manifest_create needs both builds to succeed and is then fairly fast) should`TM` prevent this. 

I can look for a maybe more save way to amend with sha's rather than tags

Reply:
@nightkr so yeah, technically that's possible... but hacky.

As much as I can come up with we would do something like this:
```
ARCH=$(arch) &&
          sha-$ARCH=$(docker images --no-trunc --quiet docker.stackable.tech/stackable/ubi8-rust-builder:latest-$ARCH) \
          echo "sha-$ARCH=$sha-$ARCH" >> $GITHUB-OUTPUT
```

and teach the `Makefile` to use those sha's rather then the tags.

I don't really like the solution, as we could mitigate the race condition with this condition:
```
  create_manifest:
    name: Create and publish manifest list
    runs-on: ubuntu-latest
    needs: ["build"]
```

WDYT?

Reply:
Yes, that would be a rework in the script. I'm not sure if I like that more than having it obvious in the pipeline tbh....